README.md

```diff
@@ -1,56 +1,34 @@
-# Ktor CLI
-Ktor CLI is a command-line tool that brings the capability to create and run [Ktor](https://ktor.io/) applications from the command line.
-
-## Install CLI tools
-
-### macOS
-
-You can install Ktor CLI tools on macOS using [Homebrew](https://brew.sh/) as follows:
-1. Add a Ktor repository using the `brew tap` command:
-   ```
-   brew tap ktorio/ktor
-   ```
-2. Install Ktor CLI tools using `brew install`:
-   ```
-   brew install --build-from-source ktor\
-   ```
-
-### Linux
-
-On Linux, you can install Ktor CLI tools using [snap](https://snapcraft.io/):
+# Ktor-cli
```

Comment:
> The `ktor-cli` tool allows generating Ktor applications..."

The executable is called just `ktor` so it can be confusing. However, I will rename the title.

Reply:
My bad. Then the correct way to write it would be "The `ktor` tool ..."